miraheze/puppet/upgrade-cp-service.py

```
@@ -0,0 +1,141 @@
+    if Debug:
+        server = f'{Debug}.wikitide.net'
+        headers['X-WikiTide-Debug'] = server
+        location = f'{domain}@{server}'
+
+        debug_access_key = os.getenv('DEBUG_ACCESS_KEY')
+
+        # Check if DEBUG_ACCESS_KEY is set and add it to headers
+        if debug_access_key:
+            headers['X-WikiTide-Debug-Access-Key'] = debug_access_key
+    up = False
+    req = requests.get(f'https://{domain}:{port}/w/api.php?action=query&meta=siteinfo&formatversion=2&format=json', headers=headers, verify=verify)
```

Ruff (0.8.2)

Undefined name `requests`

(F821)